index.html

```diff
@@ -68,6 +67,123 @@ <h2 id="support-text">I'm a Software Developer</h2>
     <img id="main-img" src="assets/illustration.svg" alt="">
   </section>
 
+  <section id="recent-works">
+    <div class="container">
+      <div class="section-title">
+        <h1 id="recent-works-title">My Recent Works</h1>
+      </div>
+      <div class="line"></div>
+      <img id="recent-works-img" src="assets/img-placeholder.png" alt="">
+      <div id="post-stories">
+        <h1>Multi-Post Stories</h1>
```

Comment:
- [ ] [OPTIONAL] There should be only one `h1` in an HTML file because of semantics and it helps search engines and web crawlers easily understand the main subject of your page. I kindly suggest that you make only your main title of "Hey there, I'm Julio" an `h1` and kindly make use of headers from `h2` down to `h6` for any other headers in your HTML file.
- [ ] [OPTIONAL] please kindly make changes and replace all other `h1` except your main title

Reply:
I replaced the h1s and left only one